client.go

```diff
@@ -140,6 +140,5 @@ func (c *Client) DoCtx(ctx context.Context, middlewares ...cliware.Middleware) (
 	req := c.Request()
 	req.SetContext(ctx)
 	req.Use(middlewares...)
-	req.Use(c.After)
```

Comment:
This indeeds seems like a duplicate, since when a request is created [here](https://github.com/delicb/gwc/blob/master/client.go#L79), all the middlewares from client are passed to request itself and this just duplicates them.